base/deprecated.jl

```diff
@@ -1601,6 +1601,11 @@ end
 # issue #6466
 # `write` on non-isbits arrays is deprecated in io.jl.
 
+function cfunction(f, r, a::Tuple)
```

Comment:
True, however I think the resulting deprecation warning message is not as helpful/clear in this case.